utils/annotations.py

```diff
@@ -10,8 +10,43 @@
 
 
 class Annotations:
-    def __init__(self, annotation_store):
-        self.annotation_store = annotation_store
+    def __init__(self, annotation_store, global_store):
+
+        if annotation_store:
+            slices = []
+            for annotation_class in annotation_store:
+                slices.extend(list(annotation_class["annotations"].keys()))
+            slices = set(slices)
+            annotations = {key: [] for key in slices}
+
+            for annotation_class in annotation_store:
+
+                for image_idx, slice_data in annotation_class["annotations"].items():
+                    for shape in slice_data:
+                        self._set_annotation_type(shape)
+                        self._set_annotation_svg(shape)
+                        # the id of the annotation is the concatenation of its rgb color
+                        id = int(
```

Comment:
The new colorpicker uses HEX colors instead of RGB so perhaps best to update this logic. Even if this won't cause an error, we could just remove the "#" from the color and use that as the id?